front/app/components/ProjectAndFolderCards/index.tsx

```diff
@@ -63,25 +68,17 @@ const ProjectAndFolderCards = ({
     trackEventByName(tracks.searchTermChanged, { searchTerm: search });
   };
 
-  const onChangeTab = (tab: PublicationTab) => {
-    setCurrentTab(tab);
-  };
-
-  if (isNilOrError(counts) || !currentTab) {
-    return null;
-  }
-
   return (
     <ProjectAndFolderCardsInner
-      currentTab={currentTab}
       statusCounts={counts}
       publicationStatusFilter={publicationStatusFilter}
       onChangeTopics={onChangeTopics}
       onChangeAreas={onChangeAreas}
-      onChangeTab={onChangeTab}
       onChangeSearch={handleSearchChange}
       showSearch={showSearch}
-      rootLevelOnly={rootLevelOnly}
+      showFilters={true}
```

Comment:
> No need to change it for this PR but I think in general hideX is a better prop. Showing something should be the default.

I rather agree with you. I called it `show` because we have other `show` parameters in this component.

> And then we should also set a default value for this, so we only need to specify the prop when we deviate from the default.

I'm not sure what is the best default. This component is used twice: one time with false, another time with true. So, the default is not obvious, and maybe it would be better if the next user will think on their own about what to choose.

Reply:
Yeah, fine to leave this one for now btw. Ideal case to me is show by default, hideX as option with false as the default value.